examples/js/loaders/SVGLoader.js

```diff
@@ -683,6 +722,253 @@ THREE.SVGLoader.prototype = {
 
 			return array;
 
+
+		}
+
+		function getNodeTransform( node ) {
+
+			if ( ! node.hasAttribute( 'transform' ) ) {
+				return null;
+			}
+
+			var transform = parseTransformNode( node );
+
+			if ( transform ) {
+
+				if ( transformStack.length > 0 ) {
+					transform.premultiply( transformStack[ transformStack.length - 1 ] );
+				}
+
+				currentTransform.copy( transform );
+				transformStack.push( transform );
+
+			}
+
+			return transform;
+	
+		}
+
+		function parseTransformNode( node ) {
+
+			var transformAttr = node.getAttribute( 'transform' );
+			var transform = null;
+			var openParPos = transformAttr.indexOf( "(" );
+			var closeParPos = transformAttr.indexOf( ")" );
+
+			if ( openParPos > 0 && openParPos < closeParPos ) {
+
+				var transformType = transformAttr.substr( 0, openParPos );
+				
+				var array = parseFloats( transformAttr.substr( openParPos + 1, closeParPos - openParPos - 1 ) );
+
+				switch ( transformType ) {
+					
+					case "translate":
+
+						if ( array.length >= 1 ) {
+
+							transform = new THREE.Matrix3();
+
+							// Translation X
+							transform.elements[ 6 ] = array[ 0 ];
```

Comment:
We generally do not access the elements directly. We use the API, instead.
```javascript
transform.set( n11, n12, n13, n21, n22, n23, n31, n32, n33 ); // first row, second row, third row
```
Or, if it is necessary to be able to read it,
```javascript
transform.set(
    n11, n12, n13,
    n21, n22, n23,
    n31, n32, n33
); 
```
Beware, the `elements` array is column-major, while the `set()` method is row-major.

Reply:
Ok I'll do it.